docs/source/install.rst

```diff
@@ -2,53 +2,51 @@
 Installation Guidelines
 =======================
 
-Dependencies
-============
-
-`NIXPy`_ is a reimplementation of the `NIX`_ library and file format for Python.
-NIXPy depends on `h5py`_, the Python interface to the HDF5 data format and numpy .
-
-The installation instructions below describe how to build or install NIXPy as a standalone, pure Python version of NIX.
-
-Dependencies:
-
-- h5py: http://docs.h5py.org/en/latest/build.html
-- numpy: https://docs.scipy.org/doc/numpy-1.10.1/user/install.html
-
-.. _NIX: https://github.com/G-Node/nix
-.. _h5py: http://www.h5py.org/
-.. _NIXPy: https://github.com/G-Node/nixpy
-
-
 Installation instructions
 =========================
 
 The latest stable release of NIXPy is available on PyPi as `nixio`_.
-Therefore, the simplest way to install NIXPy is to use pip::
+The simplest way to install NIXPy is to use pip::
 
     pip install nixio
```

Comment:
do we want to add the `pip3 install nixio` ?

Reply:
Good point.  Maybe I should also add which versions we support.  Something like
> NIXPy works with Python versions 2.7 and 3.6+ on Windows, macOS and Linux

And then we could have all pip commands duplicate
> ```
> pip3 install nixio
> ```
> or for Python 2.7
>```
> pip2 install nixio
> ```

Thoughts?

Reply:
I think that would be helpful, indeed